hxd/Window.hl.hx

```diff
@@ -380,6 +394,34 @@ class Window {
 
 	#end
 
+	function get_displayMode() : DisplayMode {
+		var mode = window.displayMode;
+
+		if ( mode == windowedMode) return Windowed;
+		if ( mode == borderlessMode) return Borderless;
+		if ( mode == fullScreenMode) return Fullscreen;

```

Comment:
Should be gated with `#if (hldx || hlsdl)`, since `window` does not exists with `usesys`.